src/OmniSharp.Roslyn.CSharp/Services/Navigation/SourceGeneratedFileService.cs

```diff
@@ -20,6 +22,9 @@ public class SourceGeneratedFileService :
         IRequestHandler<UpdateSourceGeneratedFileRequest, UpdateSourceGeneratedFileResponse>,
         IRequestHandler<SourceGeneratedFileClosedRequest, SourceGeneratedFileClosedResponse>
     {
+        private static MethodInfo? CreateDocumentIdFromSerialized => typeof(DocumentId)
```

Comment:
Should we update the EA for this? Would prefer that over readding reflection.

Reply:
@333fred Updated to remove reflection and use EA.